frontend/server/src/Exceptions/TokenDecodeException.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace OmegaUp\Exceptions;
+
+class TokenDecodeException extends \OmegaUp\Exceptions\ApiException {
```

Comment:
```suggestion
class TokenDecodeException extends Exception {
```

esta excepción no se debe mandar directamente al usuario, porque no queremos que puedan ver los claims! también, la idea es que si se atrapa esta excepción, se guarde al log los claims y se arroje otra excepción (con el mensaje `'tokenDecodeFailed'`).

Reply:
Listo.

Reply:
esto falta.

Reply:
Ahh, si lo había hecho, pero lo tuve que regresar porque PSALM se estaba quejando. Parece que ya quedó

Reply:
deja que se queje psalm. podemos arreglar los errores si salen acá en github.